client/src/components/Tag.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const TagSpan = styled.span`
+  color: white;
+  background: green;
```

Comment:
We should be looking into using the css variables that Jordan is defining at the css root so that we dont have to go through all of our code and change the values.

Reply:
Will be doing that today